conda_lock/lockfile/__init__.py

```diff
@@ -95,6 +93,9 @@ def parse_conda_lock_file(
     if not (isinstance(version, int) and version <= Lockfile.version):
         raise ValueError(f"{path} has unknown version {version}")
 
+    for p in content["package"]:
+        del p["optional"]
```

Comment:
Note to self: we need to delete the `optional` key in order to parse this as a `StrictModel`.